.github/workflows/terraform.yml

```
@@ -0,0 +1,38 @@
+name: "terraform"
+
+# for security should only run on push to bors branches (staging/trying)
+on:
+  push:
+    branches:
+      - staging
+      - trying
```

```suggestion
      - trying
    paths:
      - terraform/**/*
```
to test but it might help reduce the amount of work the CI is doing. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push